### PR TITLE
Made it look fancy

### PR DESCRIPTION
**Pull request by S-N00B-1**
If you don't like my changes Frick you

### CODE REVIEW

File: GFYPL [hunk 1]
```diff
@@ -1,12 +1,66 @@
...
+     Licensed Rights under this License. YOU has a corresponding meaning.
+  c. LICENSOR means the individual(s) or entity(ies) granting rights 
+     under this License. OWNER has a corresponding meaning
+  d. LICENCED MATERIAL means the artistic or literary work, database, 
+     or other material to which the Licensor applied this License.
+
+SECTION 2 - Licence Condition
+
+  a. USAGE
+            1. You are no permitted to use any of the licenced material
```
Comment (queer) [inline GFYPL]:
```suggestion
            1. You are not permitted to use any of the licenced material
```

File: GFYPL [hunk 2]
```diff
@@ -1,12 +1,66 @@
...
+               associated with this licence
+                     i. using or not using any of the licenced material
+                        under this licence, you are still obligated
+                        to give a responce sattisfying the terms in
```
Comment (queer) [inline GFYPL]:
```suggestion
                        to give a response satisfying the terms in
```

File: GFYPL [hunk 3]
```diff
@@ -1,12 +1,66 @@
...
+                        1(a).
+                     ii. you are also not permitted to use the
+                         licenced material as any of the following
+                                * A "Code Library"
+  b. TERMS            
+            1. You are not permitted to observe any code under
+               this licence.
+                     i. viewing or not viewing any of the licenced material
+                        under this licence, you are still obligated
+                        to give a responce sattisfying the terms in
```
Comment (queer) [inline GFYPL]:
```suggestion
                        to give a response satisfying the terms in
```

File: GFYPL [hunk 4]
```diff
@@ -1,12 +1,66 @@
...
+                        1(a).
+            2. You are not permitted to request that any of the licenced
+               material have its licence terms changed.
+                     i. requesting a change means you are obligated
+                        to give a responce sattisfying the terms in
```
Comment (queer) [inline GFYPL]:
```suggestion
                        to give a response satisfying the terms in
```

File: GFYPL [hunk 5]
```diff
@@ -1,12 +1,66 @@
...
+                        1(a).
+            3. You are not permitted to create coppies (Private or Public)
```
Comment (queer) [inline GFYPL]:
```suggestion
            3. You are not permitted to create copies (Private or Public)
```

File: GFYPL [hunk 6]
```diff
@@ -1,12 +1,66 @@
...
+               of the licenced material.
+                     i. creating a copy means you are obligated
+                        to give a responce sattisfying the terms in
```
Comment (queer) [inline GFYPL]:
```suggestion
                        to give a response satisfying the terms in
```

File: GFYPL [hunk 7]
```diff
@@ -1,12 +1,66 @@
...
+                        1(a).
+            4. You are not permitted to make an attempt to recreate
+               the exact functions of the licenced material.
+                     i. attempting to recreate the exact functions 
+                        means you are obligated to give a responce 
+                        sattisfying the terms in 1(a).
```
Comment (queer) [inline GFYPL]:
```suggestion
                        means you are obligated to give a response 
                        satisfying the terms in 1(a).
```

File: GFYPL [hunk 8]
```diff
@@ -1,12 +1,66 @@
...
+            5. It is requested by the Licensor that you do not view this licence
+                     i. viewing this licence means you are obligated
+                        to give a responce sattisfying the terms
```
Comment (queer) [inline GFYPL]:
```suggestion
                        to give a response satisfying the terms
```

**Review by queer — CHANGES_REQUESTED** [review]
Thank you for your contribution!

There were a bunch of small spelling mistakes that needed to be corrected, hence all the suggestions. However, I'm not particularly inclined to merge this unless the GFYCLA gets a similar rework.

**Comment by S-N00B-1** [comment]
In conclusion, GitHub needs spellcheck

**Comment by S-N00B-1** [comment]
I shall begin work on upgrading the Other one then!

**Comment by S-N00B-1** [comment]
The GFYCLA seems to have been fixed by [#10](https://github.com/queer/GFYPL/pull/10) As such I will unmark this as a draft.

**Review by queer — APPROVED** [review]
Just needs the conflicts fixed \^\^

**Review by valoeghese — APPROVED** [review]

**Comment by S-N00B-1** [comment]
All ready!

**Comment by queer** [comment]
Thank you for your contribution ^^

**Pull request merged**